Osiris/SDK/WeaponId.h

```diff
@@ -133,6 +136,13 @@ constexpr int getWeaponIndex(WeaponId weaponId) noexcept
     case WeaponId::Scar20: return 34;
 
     case WeaponId::Taser: return 39;
+
+    case WeaponId::Flash: return 41;
+    case WeaponId::Decoy: return 42;
+    case WeaponId::Grenade: return 43;
+    case WeaponId::IncGrenade: return 44;
+    case WeaponId::Molotov: return 45;
+    case WeaponId::Smoke: return 46;
```

Comment:
This will make aimbot / triggerbot crash. `getWeaponIndex()` function is only meant to be used in aimbot & triggerbot.